package.json

```diff
@@ -109,6 +111,7 @@
         "@codemirror/state": "0.19.9",
         "@codemirror/tooltip": "0.19.16",
         "@codemirror/view": "0.19.47",
+        "@eslint/eslintrc": "^3.0.0",
```

Comment:
This might make more sense to be in `devDependencies`?

Reply:
correct, accidentally added this to dependencies.